docs/reference/settings.rst

```diff
@@ -64,6 +64,77 @@ Override the templates used by the search front-end views.
 
 Set the number of days (default 7) that search query logs are kept for; these are used to identify popular search terms for :ref:`promoted search results <editors-picks>`. Queries older than this will be removed by the :ref:`search_garbage_collect` command.
 
+Internationalisation
+====================
+
+.. versionadded:: 2.11
+
+Wagtail supports internationalisation of content by maintaining separate trees
+of pages for each language.
+
+For a guide on how to enable internationalisation on your site, see the :ref:`configuration guide <enabling_internationalisation>`.
+
+``WAGTAIL_I18N_ENABLED``
+------------------------
+
+(boolean, default ``False``)
+
+When set to ``True``, Wagtail's internationalisation features will be enabled.
+
+For example:
+
+.. code-block:: python
+
+    WAGTAIL_I18N_ENABLED = True
+
+``WAGTAIL_CONTENT_LANGUAGES``
+-----------------------------
+
+(list, default [])
+
+A list of languages that Wagtail content can be authored in.
+
+This setting follows the same format that Django uses for its ``LANGUAGES`` setting,
+so you can set them both to the same value:
+
+.. code-block:: python
+
+    LANGUAGES = WAGTAIL_CONTENT_LANGUAGES = [
+        ('en-GB', _("English (United Kingdom)")),
+        ('en-US', _("English (United States)")),
+        ('es-ES', _("Spanish (Spain)")),
+        ('es-MX', _("Spanish (Mexico)")),
+    ]
+
+But having them separate allows you to configure many different regions on your site
+but have them share Wagtail content (but defer on things like date formatting, currency, etc):
+
+.. code-block:: python
+
+    LANGUAGES = [
+        ('en', _("English (United Kingdom)")),
+        ('en-US', _("English (United States)")),
+        ('es', _("Spanish (Spain)")),
+        ('es-MX', _("Spanish (Mexico)")),
+    ]
+
+
+    WAGTAIL_CONTENT_LANGUAGES = [
+        ('en', _("English")),
+        ('es', _("Spanish")),
+    ]
+
+This would mean that your site will respond on the URLs
+``https://www.mysite.com/es/`` and ``https://www.mysite.com/es-MX/`` but both
+of these URLs will serve content from the same "Spanish" tree in Wagtail.
```

Comment:
```suggestion
This would mean that your site will respond on the
``https://www.mysite.com/es/`` and ``https://www.mysite.com/es-MX/`` URLs, but both
of them will serve content from the same "Spanish" tree in Wagtail.
```